src/resources/payment_method.js

```diff
@@ -1,57 +1,20 @@
 import T from 'proptypes';
-import base from './base';
 
 export const propTypes = {
   id: T.string,
+  disabled_at: T.string,
```

Comment:
Nit: object should come before disabled_at

Reply:
fixed